### PR TITLE
Add manipulation routine for numpy frontend: roll

### PR DESCRIPTION
**Pull request by jucamohedano**
Implementation of roll function and its test

### CODE REVIEW

File: ivy/functional/frontends/numpy/manipulation_routines/rearranging_elements.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# local
+# global
```
Comment (Unevilicorn) [inline ivy/functional/frontends/numpy/manipulation_routines/rearranging_elements.py:1]: This should be local, since ivy is a local library

**Review by Unevilicorn — CHANGES_REQUESTED** [review]
Just a small change, I'll get this merged once this is completed

File: ivy/functional/frontends/numpy/manipulation_routines/rearranging_elements.py (hunk 1 above)
Comment (jucamohedano) [reply ivy/functional/frontends/numpy/manipulation_routines/rearranging_elements.py]: Thank you for spotting that! I made the change :)

**Comment by Unevilicorn** [comment]
Looks good, I'll get this merged!

**Pull request merged**